src/Commands/Multidev/MergeToDevCommand.php

```diff
@@ -25,9 +25,9 @@ class MergeToDevCommand extends TerminusCommand implements SiteAwareInterface
      * @param string $site_env Site & environment to merge into dev, in the form `site-name.env`.
      * @option boolean $updatedb True to update the database along with this merge
      *
-     * @usage terminus multidev:merge-to-dev <site>.<env>
+     * @usage <site>.<env>
```

Comment:
Would you change this to
```
* @usage <site>.<multidev>
```
please? It looks like this one was missed in clean-up.